MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentComponent/PaymentComponentsController.swift

```diff
@@ -49,6 +50,7 @@ public final class PaymentComponentsController: PaymentComponentsProtocol {
 
     /// Payment Component View Configuration
     public var paymentComponentConfiguration: PaymentComponentConfiguration?
+    public var paymentReviewViewController: PaymentReviewViewController?
```

Comment:
Suggestion: I won't store this viewController into the controller, I would always instantiate and pass it to the user.